index.html

```diff
@@ -9142,6 +9161,252 @@
+    void                  write(Uint8Array data);
+    void                  finish(optional Uint8Array data);
+    Promise&lt;void&gt;   waitForWritable(unsigned long amount,
+                                          optional unsigned long maxBufferedAmount);
+    Uint8Array            read();
+    void                  readInto(Uint8Array data);
+    Promise&lt;void&gt;   waitForReadableOrFinished(unsigned long amount,
+                                                    optional unsigned long maxBufferedAmount);
+    void                  close();
+                        attribute EventHandler             onacked;
+                        attribute EventHandler             onclosed;
+};</pre>
+      </div>
+    </section>
+    <section>
+      <h2>Attributes</h2>
+      <dl data-link-for="RTCQuicStream" data-dfn-for="RTCQuicStream" class=
+      "attributes">
+        <dt><dfn><code>onacked</code></dfn> of type <span class=
+        "idlAttrType"><a>EventHandler</a></span></dt>
+        <dd>
+          <p>This event handler, of event handler event type
+          <code><a>acked</a></code>, <em class="rfc2119" title="MUST">MUST</em>
+          be fired on reception of an ACK from the remote side.</p>
+        </dd>
+        <dt><dfn><code>onclosed</code></dfn> of type <span class=
+        "idlAttrType"><a>EventHandler</a></span></dt>
+        <dd>
+          <p>This event handler, of event handler event type <code>closed</code>,
+          <em class="rfc2119" title="MUST">MUST</em> be fired when the stream is
+          closed by either side.</p>
+        </dd>
+      </dl>
+    </section>
+    <section>
+      <h2>Methods</h2>
+      <dl data-link-for="RTCQuicStream" data-dfn-for="RTCQuicStream" class=
+      "methods">
+        <dt><code>write</code></dt>
+        <dd>
+          <p><dfn>write()</dfn> writes the data to local buffer and send the data when the
+          transport underneath is writeable. If the buffer is full, it will fail.</p>
+          <table class = "parameters">
+            <tbody>
+              <tr>
+                <th>Parameter</th>
+                <th>Type</th>
+                <th>Nullable</th>
+                <th>Optional</th>
+                <th>Description</th>
+              </tr>
+              <tr>
+                <td class="prmName">data</td>
+                <td class="prmType"><code>Uint8Array</code></td>
+                <td class="prmNullFalse"><span role="img" aria-label=
+                "False">&#10008;</span></td>
+                <td class="prmOptFalse"><span role="img" aria-label=
+                "False">&#10008;</span></td>
+                <td class="prmDesc"></td>
+              </tr>
+            </tbody>
+          </table>
+          <div>
+            <em>Return type:</em> <code>void</code>
+          </div>
+        </dd>
+        <dt><code>finish</code></dt>
+        <dd>
+          <p><dfn>finish()</dfn> will maybe wirte one last value before finishing and then a "fin" bit.
```

Comment:
Not "maybe write" but "write one last value (if given) and then write the QUIC fin bit"